File: pset1/hello/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    float f;

    do
    {
        f = get_float("Cash Owed: \n");
    }
    while(f < 0);

    int cents = round(f * 100);

    int coins = 0;

    while(cents != 0)
    {
        coins += cents/25;
        cents %= 25;

        coins += cents/10;
        cents %= 10;

        coins += cents/5;
        cents %= 5;

        coins += cents/1;
        cents %= 1;

        printf("Cents= %i, Coins = %i", cents, coins);
    }


}

File: pset6/cash.py
from cs50 import get_float

while True:
    change = get_float("Change owed: ")
    if change > 0:
        break

cents = round(change * 100)

total =0;

for coin in [25, 10, 5, 1]:
    total += cents//coin
    cents %= coin
submit50 cs50/problems/2020/x/sentimental/credit
print(f"Total coins: {total}")




